components/sbm-recipes-boot-upgrade/src/test/java/org/springframework/sbm/boot/upgrade_24_25/recipes/Boot_24_25_UpgradeReportRecipeTest.java

```diff
@@ -47,7 +45,8 @@ void generateReportTest() throws IOException {
         Path resultDir = RecipeIntegrationTestSupport.getResultDir(applicationDir);
         String generatedReportContent = replaceDynamicPartsInReport(resultDir.resolve("Upgrade-Spring-Boot-2.4-to-2.5.html"), resultDir);
 
-        String revision = GitSupport.getLatestCommit(resultDir.toAbsolutePath().toFile()).get().getHash();
+        GitSupport gitSupport = new GitSupport(new SbmApplicationProperties());
```

Comment:
This will disable the gitSupport fwir as the flag will be initialized with false here. I guess that's ok or maybe better.